middleware/app/Http/Controllers/Controller.php

```diff
@@ -1,11 +1,12 @@
 <?php
 
-namespace App\Http\Controllers;
+namespace App\Http\Api\V1\Controllers;
```

Comment:
@PThorpe92 I think according to PSR-0 this has to be `namespace App\Http\Controllers` to match the directory structure. I see that an identical file is in the `Api\V1` directory tho. Would this file be for non-api controllers?